src/blackdoor/cqbe/addressing/AddressTable.java

```diff
@@ -26,7 +27,16 @@
  * @version v1.0.0 - Nov 19, 2014
  */
 public class AddressTable extends ConcurrentSkipListMap<byte[], L3Address> implements Serializable, Iterable<L3Address> {
-	public static final int DEFAULT_MAX_SIZE = (int) Config.getReadOnly("default.config", "address_table_max_size");
+	public static final int DEFAULT_MAX_SIZE;
+	static{
+		int temp;
+		try{
+			temp = (int) Config.getReadOnly("default.config", "address_table_max_size");
+		}catch(ConfigFileNotFoundException e){
+			temp = 128;
+		}
+		DEFAULT_MAX_SIZE = temp;
+	}
```

Comment:
This is still nice and portable and safe, so I'll combine your changes with this when I merge, so we have both.